src/coreclr/src/tools/r2rdump/TextDumper.cs

```diff
@@ -189,8 +189,8 @@ internal override void DumpRuntimeFunction(RuntimeFunction rtf)
         /// </summary>
         internal override void DumpDisasm(RuntimeFunction rtf, int imageOffset)
         {
-            int indent = (_options.Naked ? 11 : 32);
-            string indentString = new string(' ', indent);
+            int indent = (_options.Naked ? _options.HideOffsets ? 8 : 11 : 32);
+            string indentString = new string('\t', indent / 8) + new string(' ', indent % 8);
```

Comment:
I would prefer using spaces. My editing tools have tab set to 4 spaces, some people prefer 8 spaces ...

Reply:
Agreed, fixed in 2nd commit.